test/e2e/operator_test.go

```diff
@@ -110,6 +114,7 @@ func TestVersionReporting(t *testing.T) {
 	namespacedName := types.NamespacedName{Namespace: "openshift-dns-operator", Name: "dns-operator"}
 	err = wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
 		if err := cl.Get(context.TODO(), namespacedName, deployment); err != nil {
+			t.Logf("failed to get deployment %s: %v", namespacedName, err)
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s/%s: %v", namespacedName.Namespace, namespacedName.Name, err)`